modules/dcache/src/main/java/diskCacheV111/poolManager/CostModuleV1.java

```diff
@@ -83,16 +90,61 @@ public ImmutableMap<String, String> getTagMap() {
         public PoolInfo getPoolInfo() {
             return new PoolInfo(_address, _info, _tagMap);
         }
+
+        public long getSerialId() {
+            return _serialId;
+        }
+
+        public int getTrustScore() {
+            return _trustScore;
+        }
+
+        public boolean getEnabledStatus() {
+            return _enabled;
+        }
+    }
+    public boolean getPoolStatus (String poolName) {
+        return _hash.get(poolName).getEnabledStatus();
+
     }
 
     public synchronized void messageArrived(CellMessage envelope, PoolManagerPoolUpMessage msg) {
+        int tsIncrease = 15;
```

Comment:
please add comments why this constants are selected

Reply:
comments added in [d8fd9e6](https://github.com/dCache/dcache/pull/7532/commits/d8fd9e616e6c1b8ffe68bf9691e932ead84a7663)